package NYT.NTransaction.NProto;

////////////////////////////////////////////////////////////////////////////////

message TMsgCreateTransaction
{
    required bytes TransactionId = 1;
}

message TMsgCommitTransaction
{
    required bytes TransactionId = 1;
}

message TMsgAbortTransaction
{
    required bytes TransactionId = 1;
}

////////////////////////////////////////////////////////////////////////////////
